### PR TITLE
Add a liveness probe to Valkyrie's chart

### PR DESCRIPTION
**Pull request by jakeprem**
Co-authored-by: rajones <rajones@hntb.com>

### CODE REVIEW

File: valkyrie/templates/deployment.yaml [hunk 1]
```diff
@@ -23,6 +23,13 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          livenessProbe:
+            exec:
+              command:
+                - /app/bin/valkyrie
```
Comment (jeffgrunewald) [inline valkyrie/templates/deployment.yaml]: so does this _not_ need to implement the ulimit controls to ensure it doesn't eat a ton of resoures?

Comment (rajones-hntb) [reply valkyrie/templates/deployment.yaml]: that will probably be the next step using softlimit and discuss it on entry to see if its scope for this card. We are not certain yet that performing the ping will completely address the issue `erl_child_setup closed` issue

Comment (rajones-hntb) [reply valkyrie/templates/deployment.yaml]: https://rhye.org/post/erlang-kubernetes-liveness/

File: docs/index.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ entries:
   andi:
   - apiVersion: v1
     appVersion: "1.0"
-    created: 2019-07-17T08:43:03.913273417-04:00
+    created: "2019-07-17T08:43:03.913273417-04:00"
```
Comment (jdenen) [inline docs/index.yaml]: Did you run `package.sh`? I think @smillardHNTB fixed this issue where the created field was overwritten for each package.

**Pull request merged**